Packs/LINENotify/Integrations/LINENotify/LINENotify_description.md

```diff
@@ -0,0 +1,10 @@
+### Community Contributed Integration
```

Comment:
Please remove the description from here. Its a default one. 
Write here a line about the product and how to obtain the api key

Reply:
1. Login nhttps://notify-bot.line.me/my/ with your username/password of line
2. Click 'your name' in top-right, and select "My page"
3. Click 'Generate Token' , and then input a token name and select a chat group.
4. Then click 'Generate token', then you will see a token with you. Make sure you copy the token string and keep it.